README.md

```diff
@@ -31,9 +86,21 @@ cluster](https://docs.tilt.dev/choosing_clusters.html) and example repos like
 
 `ctlptl` is a culmination of what we've learned.
 
-## How do I try it?
+## Features
+
+### Current
+
+- Docker for Mac
+- [KIND](https://kind.sigs.k8s.io/)
+- [KIND with a registry](https://kind.sigs.k8s.io/docs/user/local-registry/)
+- Allocating CPUs
```

Comment:
we only support this for D4M, right? i might nest this stuff:
```
- Docker for Mac
  - Allocating CPUs
- KIND
  - local registry
```

Reply:
KIND runs in Docker for Mac! So it totally makes sense to have KIND with minCPUs, it will just set the Docker for Mac CPUs